cpp/tests/sort/sort_test.cpp

```diff
@@ -853,9 +853,9 @@ TEST_F(SortCornerTest, WithEmptyStructColumn)
     cudf::make_structs_column(6, std::move(child_columns2), 0, rmm::device_buffer{});
   table_view input3{{struct_col3->view()}};
 
-  int_col expected3{{1, 4, 0, 2, 3, 5}};
+  int_col expected3{{4, 1, 5, 3, 2, 0}};
   auto got3 = sorted_order(input3, {order::ASCENDING});
-  CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected3, got3->view());
+  CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected3, got3->view(), debug_output_level::ALL_ERRORS);
```

Comment:
Shall we remove `debug_output_level::ALL_ERRORS`?

Reply:
sure, why not.